src/modules/firewall/tests/FirewallTests.cpp

```diff
@@ -8,52 +8,149 @@
 
 namespace tests
 {
-    static const std::string expectedFingerprint = "abc123";
-    static const std::string expectedFingerprintJson = "\"" + expectedFingerprint + "\"";
+    testing::AssertionResult JsonEq(const std::string& expectedJson, const std::string& actualJson)
+    {
+        rapidjson::Document actual;
+        rapidjson::Document expected;
 
-    class MockUtility : public GenericFirewall
+        if (expected.Parse(expectedJson.c_str()).HasParseError())
+        {
+            return testing::AssertionFailure() << "expected JSON is not valid JSON";
+        }
+        else if (actual.Parse(actualJson.c_str()).HasParseError())
+        {
+            return testing::AssertionFailure() << "actual JSON is not valid JSON";
```

Comment:
These should never happen in an unit-test. Either positive or negative the test shall pass without extra assertions.

Reply:
This is a custom assertion for comparing the JSON values returned by `Firewall::Get()`